src/components/Page-Title/Page-Title.js

```diff
@@ -0,0 +1,64 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class PageTitle extends React.Component {
+  render() {
+    let help = "";
+    let link = "";
+    let subHeader = "";
+
+   if(this.props.helpIcon){
+        help =(
+            <a className="chi-button -icon -flat -lg -bg--none -b--transparent -opacity-hover--80">
+                    <div className="chi-button__content">
+                        <i className="chi-icon icon-circle-question-outline"></i>
+                    </div>
+            </a>
+        );
+   }
+
+   if(this.props.backlink){
+       link = (
+        <a className="chi-link">
+            <div className="chi-link__content">
+                <i className="chi-icon icon-chevron-left -xs"></i>
+                <span className="-text--md">{this.props.backlinkTitle? this.props.backlinkTitle : 'Ticketing'}</span>
+            </div>
+        </a> 
+       );
+   } 
+    
+   if(this.props.subtitle){
+       subHeader = (
+        <div className="-text--md -pl--2">{this.props.subtitleText? this.props.subtitleText :'1-HB23OP (Arizona State University)'}</div>
```

Comment:
This would render the subheader always. There are cases when the subheader is not present. The entire <div> is supposed to be rendered conditionally

Reply:
subheader will be rendered only if props subtitle is set to true in boolean